hc/settings.py

```diff
@@ -21,7 +21,7 @@
 DEBUG = True
 ALLOWED_HOSTS = ['0.0.0.0','localhost', 'hc-anansi-staging.herokuapp.com',
                  'hc-anansi-production.herokuapp.com',
-                 'hc-ibutiti.herokuapp.com']
+                 'hc-ibutiti.herokuapp.com', '127.0.0.1']
```

Comment:
Why are you using this host?? I guess we have the app hosted on heroku. You should use that.

Reply:
Yes we do
but I only included that so I could be able to quickly test what am doing
It comes in handy a lot.